CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+<img src="https://atsign.dev/assets/img/@dev.png?sanitize=true">
+
+### Now for a little internet optimism
+
+# Contributing guidelines
+
+We :heart: [Pull Requests](https://help.github.com/articles/about-pull-requests/)
+for fixing issues or adding features. Thanks for your contribution!
+
+Please read our [code of conduct](code_of_conduct.md), which is based on
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](code_of_conduct.md)
+
+
+For small changes, especially documentation, you can simply use the "Edit" button
+to update the Markdown file, and start the
+[pull request](https://help.github.com/articles/about-pull-requests/) process.
+Use the preview tab in GitHub to make sure that it is properly
+formatted before committing.
+A pull request will cause integration tests to run automatically, so please review
+the results of the pipeline and correct any mistakes that are reported.
+
+If you plan to contribute often or have a larger change to make, it is best to
+setup an environment for contribution, which is what the rest of these guidelines
+describe.
+
+## Development Environment Setup
+
+
+### Prerequisites
+
+   ``` sh
+   # show how to install the tools needed to work with the code here
+   ```
+
+
+### GitHub Repository Clone
+
+To prepare your dedicated GitHub repository:
+
+1. Fork in GitHub https://github.com/atsign-foundation/at_client_sd
+2. Clone *your forked repository* (e.g., `git clone git@github.com:yourname/at_client_sd`)
```

Comment:
'k' missing at the end

Reply:
well spotted - should be fixed in latest commit

Reply:
Typo in URLs was fixed with https://github.com/atsign-foundation/at_client_sdk/pull/147/commits/5a4e16348cfc08f5a154d50782b2943010cc007a so are we good to go with this now?